src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -220,7 +220,6 @@ public void TestEvaluateOptionsProvidedAliasWithEnvVarConfig()
             string configFile = RootPath("complete.toml");
             string clientOverride = "3933d919-5ba4-4eb7-b4b1-19d33e8b82c0";
             this.fileSystem.File.WriteAllText(configFile, CompleteAliasTOML);
-            this.envMock.Setup(env => env.Get(It.IsAny<string>())).Returns((string)null);

```

Comment:
Is this line removed by mistake?

Reply:
It's not mistake. If a mock was set but never called, there would be an error.